py4DSTEM/io/datastructure/py4dstem/calibration.py

```diff
@@ -123,20 +128,25 @@ def get_Rshape(self,x):
         return shape
 
     # pixel sizes
+    @propagating_calibration
     def set_Q_pixel_size(self,x):
         self._params['Q_pixel_size'] = x
     def get_Q_pixel_size(self):
         return self._get_value('Q_pixel_size')
+
+    @propagating_calibration
     def set_R_pixel_size(self,x):
         self._params['R_pixel_size'] = x
     def get_R_pixel_size(self):
         return self._get_value('R_pixel_size')
+
     def set_Q_pixel_units(self,x):
```

Comment:
i think the `@propagating_calibration` decorator be applied to setting the R/Q pixel units as well as sizes